llvm/include/llvm/CodeGen/SelectionDAG.h

```diff
@@ -924,6 +924,20 @@ class SelectionDAG {
   /// Example: shuffle A, B, <0,5,2,7> -> shuffle B, A, <4,1,6,3>
   SDValue getCommutedVectorShuffle(const ShuffleVectorSDNode &SV);
 
+  /// Insert \p SubVec into the \p Idx sub-vector of \p Vec.
```

Comment:
Idx isn't the sub-vector but the element to insert at?
```suggestion
  /// Insert \p SubVec at the \p Idx element of \p Vec.
```